core/src/main/scripts/importer/validateData.py

```diff
@@ -1221,11 +1221,11 @@ def checkAminoAcidChange(self, value):
         # may require bundling the hgvs package:
         # https://pypi.python.org/pypi/hgvs/
         if value not in self.NULL_AA_CHANGE_VALUES:
-            # give error if a white space is found in value (this leads to errors in
+            # give error if a leading or trailing white space is found in value (this leads to errors in
             # other layers, e.g. COSMIC keywords will not be matched)
-            if ' ' in value:
+            if value.rstrip() != value:
                 # return with an error message
-                self.extra = 'White space found in amino acid change column'
+                self.extra = 'Trailing white space found in amino acid change column'
```

Comment:
Is this really a problem worth reporting? Can our java import code not handle stripping white space? If it can't we should just fix it in the java layer. I might just be misunderstanding here.